front-end/src/screens/LoginForm/index.tsx

```diff
@@ -2,141 +2,37 @@
-		const { username, password } = data;
-
-		if (username && password){
-			loginMutation({
-				variables: {
-					password,
-					username
-				}
-			}).then(({ data }) => {
-				if (data && data.login && data.login.token && data.login.user) {
-					handleLoginUser({ token: data.login.token, user: data.login.user }, currentUser);
-					history.push('/');
-				}
-			}).catch((e) => {
-				console.error('Login error', e);
-			});
-		}
-	};
-
-	const navigateAddressLogin = () => {
-		history.push('/web3login');
-	};
+	const [displayWeb2, setDisplayWeb2] = useState(true);
+	const toggleWeb2Login = () => setDisplayWeb2(!displayWeb2);
 
 	return (
 		<Grid className={className}>
 			<Grid.Column only='tablet computer' tablet={2} computer={4} largeScreen={5} widescreen={5}/>
 			<Grid.Column mobile={16} tablet={12} computer={8} largeScreen={6} widescreen={6}>
-				<Form onSubmit={handleSubmit(handleSubmitForm)}>
-					<h3>Login</h3>
-					<Form.Group>
-						<Form.Field width={16}>
-							<label>Username</label>
-							<input
-								className={errors.username ? 'error' : ''}
-								name='username'
-								placeholder='John'
-								ref={register({ maxLength:20, minLength:3, pattern: /^[A-Za-z0-9._-]*$/, required: true })}
-								type='text'
-							/>
-							{(errors.username as FieldError)?.type === 'maxLength' && <span className={'errorText'}>{messages.VALIDATION_USERNAME_MAXLENGTH_ERROR}</span>}
-							{(errors.username as FieldError)?.type === 'minLength' && <span className={'errorText'}>{messages.VALIDATION_USERNAME_MINLENGTH_ERROR}</span>}
-							{(errors.username as FieldError)?.type === 'pattern' && <span className={'errorText'}>{messages.VALIDATION_USERNAME_PATTERN_ERROR}</span>}
-							{(errors.username as FieldError)?.type === 'required' && <span className={'errorText'}>{messages.VALIDATION_USERNAME_REQUIRED_ERROR}</span>}
-						</Form.Field>
-					</Form.Group>
-
-					<Form.Group>
-						<Form.Field width={16}>
-							<label>Password</label>
-							<input
-								className={errors.password ? 'error' : ''}
-								name='password'
-								placeholder='Password'
-								ref={register({ minLength: 6 ,required: true })}
-								type='password'
-							/>
-							{errors.password && <span className={'errorText'}>{messages.VALIDATION_PASSWORD_ERROR}</span>}
-
-							<div className='text-muted'>
-								<Link to='/request-reset-password'>Forgot your password?</Link>
-							</div>
-						</Form.Field>
-					</Form.Group>
-
-					<div className={'mainButtonContainer'}>
-						<Button
-							primary
-							disabled={loading}
-							type='submit'
-						>
-							Login
-						</Button>
-					</div>
-					<div>
-						{error && <FilteredError text={error.message}/>	}
-					</div>
-					<Divider />
-					<div className={'mainButtonContainer'}>
-						<h5>Or</h5>
-						<Button
-							secondary
-							onClick={navigateAddressLogin}
-						>
-							Login With Web3 Account
-						</Button>
-					</div>
-				</Form>
+				<>
```

Comment:
I'd go for the following (simpler to read IMO), also there's no need from the `<></>` because it's already wrapped. There are only needed when you return a ReactNode with 2 different nodes (like 2 divs):
```js
				{ displayWeb2
					? <Web2Login toggleWeb2Login={toggleWeb2Login}/>
					: <Web3Login toggleWeb2Login={toggleWeb2Login}/>
				}
```